Codeinwp/neve
# Add WooCommerce Upsells

### Summary
Added the Customizer upsells.
**Note**: This PR adds the following strings:
```
Product Card
Add to Cart Button
After Image
Over Image
Category Card
Sale Tag
View more details about this
Inside
Outside
Card Content
Inline
```

### Will affect visual aspect of the product

NO

### Screenshots 
![Screenshot 2023-02-02 at 13 04 42](https://user-images.githubusercontent.com/9929553/216309131-74533c1f-04d0-4c2f-9796-2db18e7f5381.png)

### Test instructions
- Install this version of Neve
- Install WooCommerce
- Check the upsells in Customizer -> WooCommerce -> Product Catalog

### Time
~6h 27min


Closes Codeinwp/neve-pro-addon#1740.
